### DISCUSSION TITLE
Pay load size for each type of transaction in HammerDB

### DISCUSSION THREAD

**Discussion by naveenmadv** [TPC-Council/HammerDB#654] [Q&A] [answered]
I'm using HammerDB for postgres for running benchmarking tests. I was able to obtain the latency numbers for each of the transaction ctagories, NEWORD,PAYMENT,DELIVERY,SLEV,OSTAT using time profiling.

I would like to know how to check the the payload size (in kb/mb) for each of this 5 types of transactions to add more meaning to the latency number I got.

**Comment by sm-shaw** [comment, marked as answer]
It is best to use the test script to see the output from the stored procedures as the timed script by default does not print these out. This will be different from the network packet size which e.g. on SQL Server is 4KB so the actual data returned may not match up to what is transmitted. Note that by using stored procedures we are deliberately limiting the amount of data exchanged between client and server so the load is on the server side. This is explained here https://www.hammerdb.com/blog/uncategorized/why-you-should-benchmark-your-database-using-stored-procedures/
